app/services/processors/review_turnaround.rb

```diff
@@ -0,0 +1,17 @@
+module Processors
+  class ReviewTurnaround < BaseService
+    ENTITIES = ['UserProject'].freeze
+
+    def call
+      process
```

Comment:
Same comment posted in `app/services/metrics/review_turnaround/per_user_project.rb`. Is this some convention agreed upon?

Reply:
Is a way to encapsulate the `process` method

Reply:
exactly @horacio, others processors and have it like that so i wanted to keep the consistency about how are made the services. as you commented it is like a convention

Reply:
Cool!